### PR TITLE
Confused storm confusion

### PR DESCRIPTION
**Pull request by Stellation**
Ashley told me to

### CODE REVIEW

**Comment by Stellation** [comment]
Insist check the code before it gets merged

File: mods/digimon/moves.js [hunk 1]
```diff
@@ -619,9 +621,10 @@ exports.BattleMovedex = {
         basePower: 117,
         pp: 15,
         category: "Special",
-        status: "psn",
         accuracy: 100,
-        secondary: false,
+        secondary: {
+            chance: 50,
+            status: "psn",
```
Comment (HoeenCoder) [inline mods/digimon/moves.js]: your missing a closing brace

Comment (Stellation) [inline mods/digimon/moves.js]: Oops thanks!

**Comment by Stellation** [comment]
Fixed I hope (why am I so bad at this coding thing?)

**Comment by DeathlyPlays** [comment]
That isn't Travis build format :/

**Comment by Stellation** [comment]
AAAARRRGGHH! Oh well, :T ill fix it ig

**Comment by Stellation** [comment]
Better?

File: mods/digimon/moves.js (hunk 1 above)
Comment (DeathlyPlays) [inline mods/digimon/moves.js]: Yup now you got that closing brace, it's looking better :D

**Pull request merged**